test/misc/update.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
```

Comment:
Let's not write any more old-style tests. Instead, add a new test (using Mocha) to https://github.com/tmpvar/jsdom/blob/master/test/jsdom/misc.js